demosplan/DemosPlanCoreBundle/Logic/Statement/AssessmentTableExporter/Enum/ListLineWidth.php

```diff
@@ -15,7 +15,7 @@
 enum ListLineWidth: int
 {
     case VERTICAL_SPLIT_VIEW = 7;
-    case VERTICAL_NOT_SPLIT_VIEW = 17;
+    case VERTICAL_NOT_SPLIT_VIEW = 18;
     case HORIZONTAL_SPLIT_VIEW = 12;
-    case HORIZONTAL_NOT_SPLIT_VIEW = 24;
+    case HORIZONTAL_NOT_SPLIT_VIEW = 27;
```

Comment:
These enum entries need to keep their values.
This is because STN texts could contain lists - and lists need extra space for their intendation
[Export-81.pdf](https://github.com/user-attachments/files/17279905/Export-81.pdf)

Reply:
weird, i already tested it with a list and i didn,t get that, i tried it again with all export and it seems  in order, here are the exports :
[Export-251.pdf](https://github.com/user-attachments/files/17282601/Export-251.pdf)
[Export-252.pdf](https://github.com/user-attachments/files/17282625/Export-252.pdf)
[Export-253.pdf](https://github.com/user-attachments/files/17282627/Export-253.pdf)

Reply:
I see now what u mean, i had to export all statements to see that. I thought may be i can use the list width to adjust the text width but i have to do it separately i think

Reply:
i fixed it here https://github.com/demos-europe/demosplan-core/pull/3789/files